### ISSUE TITLE
Failed unit tests- remote linux server

### ISSUE THREAD

**Issue by noorvanbeers** [Tudat/tudat#664]
Dear Dominic,

I have downloaded, installed, and compiled the tudat bundle as per the linux server instructions on the tudat website, however when I run the unit tests they cannot be found. I updated the tudat through git, as this was the advice for another failed unit tests issue, and then recompiled but the issue unfortunately persists. The server I am using is the Thales linux server, unfortunately I do not have more information regarding the operating system.

[FailedTestsLog.txt](https://github.com/Tudat/tudat/files/6295020/FailedTestsLog.txt)
[CompilationLog.txt](https://github.com/Tudat/tudat/files/6295019/CompilationLog.txt)

**Comment by DominicDirkx** [comment]
Hi Noor,

It seems that you have only run cmake, but have not built the actual libraries/tests. Is this correct?

If, in your build folder, you run the make command, the compilation should get going,

Dominic

p.s. As you may have seen on our website, we have a new tudat setup, including a python interface. If you let me know what you're planning to use tudat for, I can let you know how mature the python interface is for your purposes

**Comment by noorvanbeers** [comment]
Hi Dominic,

I realised as soon as I posted the issue that I had not built the libraries.... but I thought I'd wait until it was all up and running to update the issue, my apologies! It's built and all the unit tests passed.

I am aware that there is a python interface which I would have loved to use but I'd already written a bulk of code in cpp when I heard! So I decided to continue in cpp. I'm using it for an MDO which is my thesis topic.
